tox.ini

```diff
@@ -0,0 +1,10 @@
+[tox]
```

Comment:
So do we still now use `tox`? I don't see any references to it in any of the new additions -- it should probably be removed if we don't need it.

Reply:
We no longer use `tox` and thus, yes, this dependency should be deleted. Done.